src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/dataPrivacyRulesForm/sourceSuggestionExamples.tsx

```diff
@@ -0,0 +1,78 @@
+import Modal from 'react-bootstrap/lib/Modal';
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import Button from 'app/components/button';
+
+
+type Props = {
+  examples: Array<string>;
+  sourceName: string;
+}
+
+type State = {
+  isOpen: boolean;
+}
+
+class SourceSuggestionExamples extends React.Component<Props, State> {
+  state: State = {
+    isOpen: false
+  };
+
+  toggleModal = () => {
+    this.setState({isOpen: !this.state.isOpen});
+  };
+
+  stopPropagation = (e) => {
+    // Necessary to stop propagation of click events from modal that we can't
+    // catch otherwise.
+    e.stopPropagation();
+  }
+
+  render() {
+    const {isOpen} = this.state;
+    const {examples, sourceName} = this.props;
+
+    if (examples.length == 0) {
+      return (<span></span>);
```

Comment:
I would return null here, but better yet... if examples.length === 0 this component should not be rendered